cmd/hcdev/hcdev.go

```diff
@@ -498,6 +504,7 @@ func setupApp() (app *cli.App) {
 							"-execpath="+filepath.Join(rootExecDir, roleName),
 							"-port="+strconv.Itoa(freePort),
 							"-mdns=true",
+							"-nat-upnp=true",
```

Comment:
this line says -nat  but the flag is -no-nat.  It needs to agree!

Reply:
yes, will change.